backend/satellite_tools/reposync.py

```diff
@@ -734,6 +735,17 @@ def import_updates(self, plug):
             elif notices_type == 'patches':
                 self.upload_patches(notices)
 
+    @staticmethod
+    def _getDecompressedFileChecksum(abspath, hashtype):
```

Comment:
Let's try to prevent introducing more camelCase code in Python files if possible :smile: 

Better use something more pythonic like, `_get_decompressed_file_checksum`

Reply:
Makes sense. I was confused by the `getFileCheckSum`, but that one is imported from somewhere else.